azure/image/azure-ubuntu.pkr.hcl

```diff
@@ -7,11 +7,20 @@ source "azure-arm" "ubuntu" {
   vm_size                           = "${var.vm_size}"
   os_type                           = "Linux"
   image_publisher                   = "Canonical"
-  image_offer                       = "0001-com-ubuntu-minimal-jammy"
-  image_sku                         = "minimal-22_04-lts"
+  image_offer                       = "0001-com-ubuntu-confidential-vm-jammy"
+  image_sku                         = "22_04-lts-cvm"
   managed_image_name                = "${var.az_image_name}"
   managed_image_resource_group_name = "${var.resource_group}"
   build_resource_group_name         = "${var.resource_group}"
+
+  shared_image_gallery_destination {
+    subscription         = "${var.subscription_id}"
+    resource_group       = "${var.resource_group}"
+    gallery_name         = "caaubntcvmsGallery"
+    image_name           = "cc-image"
+    image_version        = "0.0.1"
```

Comment:
I think parametrising the image_name and image_version will be good.

Reply:
Make sense, will do it